package.json

```diff
@@ -39,7 +39,7 @@
     "zone.js": "~0.10.2"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "~0.1001.6",
+    "@angular-devkit/build-angular": "^0.1002.0",
```

Comment:
Would it be possible to revert this change and the package-lock change please? Thanks!

Reply:
Yes I reverted the commit and made changes to just financial-aid.component.ts